website/versioned_docs/version-v1.x/installation.md

```diff
@@ -90,32 +107,46 @@ circuits. For more details on which artifacts have undergone a trusted setup, pl
 
 Note the locations of the `.zkey` files as the CLI requires them as command-line flags.
 
+**Download test artifacts**
+
+```bash
+pnpm download:test-zkeys
+```
+
+**Download ceremony artifacts**
+
+```bash
+pnpm download:ceremony-zkeys
+```
+
 ### Generate `.zkey` files
 
-If you wish to generate `.zkey` files from scratch, first navigate to `cli/`
-and edit `zkeys.config.yml`. Set the parameters you need.
+If you wish to generate `.zkey` files from scratch, first navigate to `circuits/circom`
+and edit `circuits.json`. Set the parameters you need.
 
 Next, run the following to compile the circuits with parameters you specified:
 
+**for the c++ witness generator**
+
 ```bash
-pnpm exec zkey-manager compile -c zkeys.config.yml
+pnpm test:circuits-c
 ```
 
-Next, download the `.ptau` file:
+\*_for the wasm witness generator_
```

Comment:
I think it should look like L129? Since they are the same level.
So it might be `**for the wasm witness generator**`

Reply:
yup thanks, updating now!